### ISSUE TITLE
Parameterized Constructor constraints

### ISSUE THREAD

**Issue by YairHalberstadt** [dotnet/csharplang#1574]
You can currently put a constraint on a generic type that it must have a parameterless constructor:

```
public class A <T> where T : new()
```
Should it be possible to define parameterised constructors as so:

```
public class A <T> where T : new(int, string)
```

Which would mean that T has to have a constructor taking an int and a string in that order

**Comment by DavidArno** [comment]
Duplicate of #769. Also, covered by plans around #110.

**Comment by dzmitry-lahoda** [comment]
Close?

**Comment by YairHalberstadt** [comment]
Closed as duplicate of #769
